aiida_quantumespresso/parsers/parse_xml/pw/parse.py

```diff
@@ -252,20 +253,21 @@ def parse_pw_xml_post_6_2(xml):
         'time_reversal_flag': time_reversal,
         'symmetries': symmetries,
         'lattice_symmetries': lattice_symmetries,
-        'do_not_use_time_reversal': inputs['symmetry_flags']['noinv'],
+        'do_not_use_time_reversal': inputs.get('symmetry_flags', {}).get('noinv', None),
         'spin_orbit_domag': outputs['magnetization']['do_magnetization'],
         'fft_grid': [value for _, value in sorted(outputs['basis_set']['fft_grid'].items())],
         'lsda': lsda,
         'number_of_spin_components': nspin,
-        'no_time_rev_operations': inputs['symmetry_flags']['no_t_rev'],
+        'no_time_rev_operations': inputs.get('symmetry_flags', {}).get('no_t_rev', None),
         'inversion_symmetry':
         inversion_symmetry,  # the old tag was INVERSION_SYMMETRY and was set to (from the code): "invsym    if true the system has inversion symmetry"
         'number_of_bravais_symmetries': nrot,  # lattice symmetries
         'number_of_symmetries': nsym,  # crystal symmetries
-        'wfc_cutoff': inputs['basis']['ecutwfc'] * hartree_to_ev,
+        'wfc_cutoff': inputs.get('basis', {}).get('ecutwfc', -1.0) * hartree_to_ev,
```

Comment:
Are there really cases where ecutwfc is not defined?

Reply:
This should be the issue with the still missing part of cp's xml file